src/functionalTest/resources/application-test.yaml

```diff
@@ -7,15 +7,24 @@ etl:
   connection-string: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:52189/devstoreaccount1;"
 
 extraction:
+  max:
+    batch:
+      row: 1
+  filter:
+    jurisdiction:
+      in:
```

Comment:
Jurisdictions to extract filter, this is to maintain backward compatibility  and avoid not initialised caseTypes